templates/ces_client/about.html

```diff
@@ -0,0 +1,74 @@
+{% extends "base.html" %}
+{% load static %}
+{% load ces_client_extras %}
+{% block title %}Connect{% endblock %}
+
+{% block extra_css %}
+    <link rel="stylesheet" href="{% static 'css/botui.min.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/botui-theme-default.css' %}" />
+{% endblock %}
+
+{% block content %}
+<div class='row'>
+    <div class='col-10 offset-1 mt-5 mb-3 pb-3'>
+        <div class='row mb-3'>
+            <div class='col-lg-2 pr-0'>
+                <a class="img-fluid float-left mr-0" href="{% url 'index' %}"><img src="{% static 'images/CESlogo.svg' %}" width='100'></a>
+            </div>
+            <div class='col'>
+                <h3 class='mt-4'>Connect to End Homelessness</h3>
+                <h4>A unified community response</h4>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row'>
+    <div class='col-10 offset-1 mb-3 pb-3'>
+        <h5><strong>About Coordinated Entry</strong></h5>
+        <p>Coordinated Entry is the front door to connect people facing homelessness to housing and supports in an accessible, equitable, transparent manner. This allows all partners to collaborate to improve access and outcomes for people without permanent housing while delivering trauma informed services.</p>
+        <p>People facing homelessness are asked to complete a Housing Assessment and linked to housing, if eligible, as opportunities become available.</p>
+
+        <h5 class='mt-5'><strong>Shelter Access in Cook County Suburbs</strong></h5>
+        <p>View shelter options and next steps <a href="http://www.suburbancook.org/emergency" target="_blank">by clicking here</a>.</p>
+
+        <h5 class='mt-5'><strong>Shelter Access In Chicago</strong></h5>
+        <p>Please visit any hospital emergency room or police station in Chicago, or call 311 and request transportation to shelter.</p>
+        <p>
+            <strong><i class="fas fa-user"></i> Single individuals can request shelter in person at the Chicago Garfield Service Center.</strong><br> 
```

Comment:
i would suggest changing these to side-by-side cols in a row. i would have the icon on top of the text too.